interface/modules/custom_modules/oe-module-comlink-telehealth/Readme.md

```diff
@@ -0,0 +1,26 @@
+### Comlink Telehealth Module
+
+This project implements the Comlink Telehealth system for the Comlink OpenEMR installation.  
+
+### Installing Module
```

Comment:
Can this step now be removed (assuming no need for composer now that is in codebase)?

Reply:
Removed this step and added more documentation information.